scripts/setup/setup-certbot

```diff
@@ -96,9 +109,13 @@ symlink_with_backup() {
     ln -nsf "$1" "$2"
 }
 
-CERT_DIR=/etc/letsencrypt/live/"$DOMAIN"
-symlink_with_backup "$CERT_DIR"/privkey.pem /etc/ssl/private/zulip.key
-symlink_with_backup "$CERT_DIR"/fullchain.pem /etc/ssl/certs/zulip.combined-chain.crt
+if [ -z "$deploy_hook" ]; then
+  # If not deploy hook was specified, assume we're deploying to the default
+  # location Zulip wants.
+  CERT_DIR=/etc/letsencrypt/live/"$DOMAIN"
+  symlink_with_backup "$CERT_DIR"/privkey.pem /etc/ssl/private/zulip.key
+  symlink_with_backup "$CERT_DIR"/fullchain.pem /etc/ssl/certs/zulip.combined-chain.crt
```

Comment:
We do 4-space indent in shell scripts (see the rest of the file)

Reply:
Fixed.